app/assets/stylesheets/nucore.scss

```diff
@@ -408,3 +409,133 @@ textarea.wide {
   padding: 0.4em;
   box-sizing: border-box;
 }
+
+body {
+  background: #ffffff; /* Old browsers */
+  background: -moz-linear-gradient(top, #ffffff 53%, #e4e4e4 100%); /* FF3.6-15 */
+  background: -webkit-linear-gradient(top, #ffffff 53%,#e4e4e4 100%); /* Chrome10-25,Safari5.1-6 */
+  background: linear-gradient(to bottom, #ffffff 53%,#e4e4e4 100%); /* W3C, IE10+, FF16+, Chrome26+, Opera12+, Safari7+ */
+  filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#ffffff', endColorstr='#e4e4e4',GradientType=0 ); /* IE6-9 */
+}
+.manage-mode {
+  background: #ffffff; /* Old browsers */
+  background: -moz-linear-gradient(top, #ffffff 53%, $lightBlue 100%); /* FF3.6-15 */
+  background: -webkit-linear-gradient(top, #ffffff 53%,$lightBlue 100%); /* Chrome10-25,Safari5.1-6 */
+  background: linear-gradient(to bottom, #ffffff 53%, $lightBlue 100%); /* W3C, IE10+, FF16+, Chrome26+, Opera12+, Safari7+ */
+  filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#ffffff', endColorstr='$lightBlue',GradientType=0 ); /* IE6-9 */
+}
+html,body{
+  min-height: 100%;
+}
+
+body nav .navbar-inner .container .nav, .nav-tabs{
+  border-bottom: 1px solid $orange;
+  min-height: 40px;
+}
+
+.manage-mode nav .navbar-inner .container .nav, 
+.manage-mode .nav-tabs {
+  border-color: $blue;
+}
+
+.alert-info {
+  background-color: transparent;
+  border-color: transparent;
+  color: $blue;
+  font-size: 24px;
+  line-height: 1em;
+}
+
+.navbar-inner {
+  box-shadow: none !important;
+  background: none;
+  border: none;
+}
+
+.brand{
+  font-family: inherit !important;
+}
+
+.nav li:hover{
+  border: none !important;
+  margin-top: 0 !important;
+}
+
+.nav li a{
+  padding: 13px;
+  text-decoration: none;
+  color: black;
+  transition: ease 0.3s all;
+}
+.nav li.active{
+  border:none !important;
+  margin-top: 0 !important;
+  margin-bottom: -1px;
+}
+
+.nav li.active a{
+  border: 1px solid $orange !important;
+  border-radius: 4px 4px 0 0;
+  border-bottom-color: transparent !important;
+  background-color: white !important;
+  box-shadow: none !important;
+}
+
+.manage-mode .nav li.active a{
+  border-color: $blue !important;
```

Comment:
Can we use `$useColor` here rather than `$blue` so it can be overridden easier? Same with `$manageColor` for `$orange`?